packages/sanity/src/core/studio/StudioProvider.tsx

```diff
@@ -59,7 +59,9 @@ export function StudioProvider({
       <StudioTelemetryProvider config={config}>
         <LocaleProvider>
           <ResourceCacheProvider>
-            <TasksProvider>{children}</TasksProvider>
+            <TasksSetupProvider>
```

Comment:
What do you think about renaming this to something reusable with comments? I heard and interesting name when the name for the datasets was in discussion. 
CollaborativeSetupProvider?

Reply:
I have something like that in mind yes. Just wondering what to name it, since it is "workflow" metadata related, and you can argue that the whole studio is Collaborative.

We can put it to the slack channel?

Reply:
Sure! Let's bring it to slack